mrtt-ui/package.json

```diff
@@ -10,7 +10,7 @@
     "@mui/icons-material": "^5.6.2",
     "@mui/material": "^5.6.4",
     "@mui/styled-engine-sc": "^5.6.1",
-    "@mui/x-date-pickers": "^5.0.0-alpha.2",
+    "@mui/x-date-pickers": "5.0.0-alpha.2",
```

Comment:
Lock the date pickers to a version incase the api and behaviour changes in the future. Want to reduce the occurrence of unexpected bugs from this.